instagram/src/App.js

```diff
@@ -3,32 +3,34 @@ import dummyData from './dummy-data';
 import PostContainer from './components/PostContainer/PostContainer.js';
 import SearchBar from './components/SearchBar/SearchBar.js';
 import './App.css';
-import PropTypes from 'prop-types';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      posts: dummyData,
+      posts: [],
       searchInfo: ''
     }
   }
 
+  componentDidMount() {
+    this.setState({posts: dummyData})
+  }
+
   searchItemHandler =(event)=> {
     this.setState({searchInfo: event.target.value});
   }
 
   searchSubmitHandler=(event)=> {
     event.preventDefault();
-    let myPosts= this.state.posts.filter(item => item.username === this.state.searchInfo);
-    console.log(myPosts);
+    let myPosts= this.state.posts.filter(item => item.username.toLowerCase() === this.state.searchInfo.toLowerCase());
```

Comment:
I'm trying to understand the reasoning behind the calls of `toLowercase()` here.

Reply:
after you declared `item` as a function an if statement checking if the username `includes()` the `target` value then return the `item` would work here as well. It works, and is an interesting way of solving the problem, but may not be ideal. Great work!